sql/core/src/test/scala/org/apache/spark/sql/ColumnExpressionSuite.scala

```diff
@@ -921,6 +922,132 @@ class ColumnExpressionSuite extends QueryTest with SharedSparkSession {
     }
   }
 
+  private def captureStdOut(block: => Unit): String = {
+    val capturedOut = new ByteArrayOutputStream()
+    Console.withOut(capturedOut)(block)
+    capturedOut.toString()
+  }
+
+  test("explain") {
```

Comment:
Hm, I wouldn't add a bunch of tests though. `expr.sql` isn't actually strictly providing the compatibility in its string representation, and this test case potentially fails often.

Reply:
yeah that was what I was wondering if this output is expected stable enough. If not then probably we should not add such tests.

Reply:
> Hm, I wouldn't add a bunch of tests though. `expr.sql` isn't actually strictly providing the compatibility in its string representation, and this test case potentially fails often.

In fact, explain not only reactor `expr.sql`, and `expr.toString` too.

Reply:
shall we use golden file tests? We don't expect it to be stable but we should be aware if it gets changed.

Reply:
+1 if we still want be aware it's changing, golden files is a better way.